tasks/synth.py

```diff
@@ -82,3 +82,42 @@
     "\g<1>if hasattr(\g<5>, 'name'):\n"
     "\g<1>    \g<2>\g<3>    \g<4>\g<5>\g<6>    \g<7>"
 )
+
+
+# fix the combined shared/local modules. 
+# https://github.com/GoogleCloudPlatform/google-cloud-python/pull/5364
+# https://github.com/googleapis/gapic-generator/issues/2058
+s.replace(
+    "google/cloud/*/types.py",
+    "for module in \(\n(.*\n)*?\):\n(    .*\n)+",
+    """_shared_modules = [
+    http_pb2,
+    iam_policy_pb2,
+    policy_pb2,
+    any_pb2,
+    descriptor_pb2,
+    duration_pb2,
+    empty_pb2,
+    field_mask_pb2,
+    timestamp_pb2,
+    status_pb2,
+]
+
+_local_modules = [
+    cloudtasks_pb2,
+    queue_pb2,
+    target_pb2,
+    task_pb2,
+]
+
+for module in _shared_modules:
+    for name, message in get_messages(module).items():
+        setattr(sys.modules[__name__], name, message)
+        names.append(name)
+
+for module in _local_modules:
+    for name, message in get_messages(module).items():
+        message.__module__ = 'google.cloud.tasks_v2beta2.types'
+        setattr(sys.modules[__name__], name, message)
+        names.append(name)
+""")
```

Comment:
Ugh, can we not just fix the generator?

Reply:
OK, I see the [generator issue](https://github.com/googleapis/gapic-generator/issues/2058).